externs/ngeox.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @type {Object}
+ */
+var ngeox;
+
+/**
+ * Namespace.
+ * @type {Object}
+ */
+ngeox.interaction;
+
+/**
+ * Interactions for measure tools base class.
+ * @typedef {{
+ *    startMsg: (string|undefined),
+ *    style:(ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined)
+ * }}
+ */
+ngeox.interaction.MeasureBaseOptions;
+
+/**
+ * Interactions for measure tools.
+ * @typedef {{
+ *    startMsg: (string|undefined),
+ *    continueMsg: (string|undefined),
+ *    style: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined),
+ *    sketchStyle: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined)
+ * }}
+ */
+ngeox.interaction.MeasureOptions;
```

Comment:
This typedef need not be in ngeox.js. It could be in measure.js, before the ngeo.interaction.Measure constructor.